### PR TITLE
Semester statistics

### PR DESCRIPTION
**Pull request by janno42**
adds semester statistics and refactors the progress bars

### CODE REVIEW

File: evap/staff/templates/staff_semester_view.html [hunk 1]
```diff
@@ -11,75 +12,122 @@
...
-
-            <div class="tab-content">
-            {% for state, courses in courses_by_state %}
-                <div id="tab{{ forloop.counter }}" class="tab-pane">
-                <table class="table table-striped">
+        <div class="panel-body">
+            <table class="table">
+                <thead>
+                    <tr>
+                        <th class="col-sm-3">{% trans "Evaluation periods" %}</th>
```
Comment (karyon) [inline evap/staff/templates/staff_semester_view.html]: there is a plural here...?

Comment (janno42) [inline evap/staff/templates/staff_semester_view.html]: "the evaluation periods of the courses are from ... to ...". but of course one could also say "the evaluation period of the semester is from ... to ...".
i'll change it :)

File: evap/staff/templates/staff_semester_view.html [hunk 2]
```diff
@@ -11,75 +12,122 @@
...
+                        <th class="col-sm-3">{% trans "Finished evaluations" %}</th>
```
Comment (karyon) [inline evap/staff/templates/staff_semester_view.html]: maybe "evaluated courses"?

File: evap/staff/templates/staff_semester_view.html [hunk 3]
```diff
@@ -11,75 +12,122 @@
...
-                        {% include_staff_semester_view_course semester state course disable_if_archived %}
+                        <td>{{ first_start|date:"SHORT_DATE_FORMAT" }} - {{ last_end|date:"SHORT_DATE_FORMAT" }}</td>
+                        <td>
+                            {% include_progress_bar num_courses_evaluated num_courses %}
+                        </td>
+                        <td>
+                            {% include_progress_bar num_comments_reviewed num_comments %}
+                        </td>
+                        <td>
+                            {% include_progress_bar num_votes num_enrollments_in_evaluation %}
```
Comment (karyon) [inline evap/staff/templates/staff_semester_view.html]: actually here the percentage is more interesting than the absolute numbers. but having the three progress bars there looks really cool...

still, could you show the percentage in plain sight?

File: evap/staff/templates/staff_semester_view.html (hunk 2 above)
Comment (janno42) [inline evap/staff/templates/staff_semester_view.html]: check

**Comment by janno42** [comment]
now with completely new progress bars

**Comment by karyon** [comment]
yes i like

**Pull request merged**